<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionwindow</class>
 <widget class="QDialog" name="optionwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0,81,44);
item {
background-color: transparent;
}
item:selected {
background-color: rgb(163, 3, 47);
}</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 13pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
 background-color: rgb(230,230,230);
    color: black;
    font: bold;</string>
   </property>
   <item>
    <property name="text">
     <string>Off</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>On</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
 background-color: rgb(230,230,230);
    color: black;
    font: bold;</string>
   </property>
   <property name="text">
    <string>Queen of Spades Mode</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
font: bold;
font-size: 10px;
color: lightgrey</string>
   </property>
   <property name="text">
    <string>Queen of Spades: +5 cards and skip a turn. Valued at 50 points.</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>95</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 12pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
 background-color: rgb(230,230,230);
    color: black;
    font: bold;</string>
   </property>
   <property name="text">
    <string>Game mod up to X points</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
font: bold;
font-size: 10px;
color: lightgrey</string>
   </property>
   <property name="text">
    <string>Game over if someone has more than 125 points. 125 nullified.</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>95</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 13pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
 background-color: rgb(230,230,230);
    color: black;
    font: bold;</string>
   </property>
   <item>
    <property name="text">
     <string>125</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>225</string>
    </property>
   </item>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>170</y>
     <width>361</width>
     <height>151</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;;
    font-size: 9pt;
    border: 1px solid;
    border-color: rgb(192,192,192); 
    border-radius: 3px; 
    color: lightgrey;
    font: bold;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;*****Rules*****&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; I'm sure a lot of people have played this game.  She has many names and even more rules.  We will use the following:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; 1.&lt;span style=&quot; font-weight:400;&quot;&gt; The goal is to throw off all the cards as quickly as possible, leaving opponents with the maximum number of cards&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; &lt;/span&gt;2.&lt;span style=&quot; font-weight:400;&quot;&gt; For each card at the end of the game, points are awarded - 10 for ten, queen, king, 15 for ace, 20 for jack.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; &lt;/span&gt;3.&lt;span style=&quot; font-weight:400;&quot;&gt; The first one who jumped the bar of 125 or 225 points loses.  The one who scores exactly 125 or 225 points - they burn out and he starts from scratch (lucky)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; &lt;/span&gt;4.&lt;span style=&quot; font-weight:400;&quot;&gt; The player must lay down a card (or several) of the same value, or the same suit (except for exceptions)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; &lt;/span&gt;5.&lt;span style=&quot; font-weight:400;&quot;&gt; The six must be covered - we take cards from the deck until we cover&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; &lt;/span&gt;6.&lt;span style=&quot; font-weight:400;&quot;&gt; Seven forces the opponent to draw a card&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; &lt;/span&gt;7.&lt;span style=&quot; font-weight:400;&quot;&gt; Eight - two cards and skip a turn&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; &lt;/span&gt;8. &lt;span style=&quot; font-weight:400;&quot;&gt;Jack - can fall on any suit and you can order any suit&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; &lt;/span&gt;9. &lt;span style=&quot; font-weight:400;&quot;&gt;Ace - the next player skips a turn&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;10. &lt;span style=&quot; font-weight:400;&quot;&gt;Finished the game with a jack: -20 points, two jacks: -40, etc.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
